monailabel/utils/others/generic.py

```diff
@@ -199,6 +200,17 @@ def device_list():
     return devices
 
 
+def device_list_gpu():
```

Comment:
why can't we fix `device_list` method instead of creating new `device_list_gpu`?

can you share the output difference between `device_list` and `device_list_gpu` methods?